Index.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body id="main">
+    <titile id="title">
+        Dr. Norman Borlaug
+        </title>
+        <div id="img-div">
+          <img id="image" src="ADD YOUR SOURCE" > </img>
```

Comment:
![review](https://user-images.githubusercontent.com/109443842/200289425-0b83cc0e-ebcd-4257-8ad4-7bb0519a8b5a.jpg)
I think it will be nice if you actually have an image of the person that you are documenting by typing the name of the file or the link to the photo followed by the file name like `<img src="image_folder\name_of_photo" alt="image">`  so that it will be easier for anyone who is reading it to know the person better.